### PR TITLE
add tagged-release script

### PR DESCRIPTION
**Pull request by atonks2**
Created a "demo" repo to test this script: https://github.com/atonks2/demo/tags

### CODE REVIEW

**Review by adamdecaf — APPROVED** [review]

**Review by vxio — APPROVED** [review]
nice checks in the script 👍🏻

File: tagged-release.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+fi
+
+expectedHeader=$(printf "## $1 (Released %s)" "$(date +"%Y-%m-%d")")
+if [ "$firstLine" != "$expectedHeader" ]
+then
+  echo "Did you update the CHANGELOG's header? Expected \"$expectedHeader\", found \"$firstLine\""
+  exit
+fi
+
+# see https://github.com/moovfinancial/engineering-guide#open-source-releases
```
Comment (vxio) [inline tagged-release.sh:48]: Oops, I missed this earlier. We should remove this comment since the link is private.

**Comment by atonks2** [comment]
Good catch. I’ll remove it.

**Comment by adamdecaf** [comment]
IMO it's fine to link to the engineering guide. It's not really a secret we have a guide for writing code.

**Pull request merged**